client/lib/abtest/active-tests.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
 		allowExistingUsers: true,
 	},
 	coldStartReader: {
-		datestamp: '20160804',
+		datestamp: '20160824',
 		variations: {
 			noEmailColdStart: 20,
```

Comment:
@gibrown for the `EmailColdStart` variant, do you mean noEmailColdStart + autofollows?